### ISSUE TITLE
Typescript error on ActivityIndicator

### ISSUE THREAD

**Issue by kickbk** [callstack/react-native-paper#2464]
"react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
"react-native-paper": "^4.5.0",
"@expo/vector-icons": "^12.0.2",

Using your snack example in TS project:
`<ActivityIndicator animating={true} color="blue" />`
Gives TS error:
```
Type '{ animating: true; color: string; }' is missing the following properties from type 'Pick<Props, "ref" | "hitSlop" | "onLayout" | "pointerEvents" | "removeClippedSubviews" | "style" | "testID" | "nativeID" | "collapsable" | "needsOffscreenAlphaCompositing" | ... 49 more ... | "hidesWhenStopped">': accessibilityComponentType, accessibilityTraits
```

**Comment by kickbk** [comment]
Consider updating the bot ;)

**Comment by MohammedFaragallah** [comment]
related to https://github.com/callstack/react-native-paper/issues/2442

**Comment by MohammedFaragallah** [comment]
fixed by #2522
